ros2service/ros2service/api/__init__.py

```diff
@@ -34,6 +37,48 @@ def get_service_names(*, node, include_hidden_services=False):
     return [n for (n, t) in service_names_and_types]
 
 
+def get_service_class(node, service, blocking=False, include_hidden_services=False):
+    srv_class = _get_service_class(node, service, include_hidden_services)
+    if srv_class:
+        return srv_class
+    elif blocking:
+        print('WARNING: service [%s] does not appear to be started yet' % service)
+        while rclpy.ok():
+            srv_class = _get_service_class(node, service, include_hidden_services)
+            if srv_class:
+                return srv_class
+            else:
+                sleep(0.1)
+    else:
+        print('WARNING: service [%s] does not appear to be started yet' % service)
+    return None
+
+
+def _get_service_class(node, service, include_hidden_services):
+    service_names_and_types = get_service_names_and_types(
+        node=node,
+        include_hidden_services=include_hidden_services)
+
+    service_type = None
+    for (service_name, service_types) in service_names_and_types:
+        if service == service_name:
+            if len(service_types) > 1:
+                raise RuntimeError(
+                    "Cannot echo service '%s', as it contains more than one type: [%s]" %
+                    (service, ', '.join(service_types))
```

Comment:
IMO, f-strings are more readable
```suggestion
                    f"Cannot echo service '{service}', as it contains more than one type: [{', '.join(service_types)}]")
```